web3/net/service.go

```diff
@@ -0,0 +1,36 @@
+package net
+
+import (
+	"fmt"
+	"github.com/Oneledger/protocol/service"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"math/big"
+)
+
+// PublicNetAPI is the eth_ prefixed set of APIs in the Web3 JSON-RPC spec.
+type PublicNetAPI struct {
+	networkVersion string
+	peerCount      hexutil.Big
+}
+
+// NewService creates an instance of the public Net Web3 API.
+func NewService(clientCtx *service.Context) *PublicNetAPI {
+	peerCount := big.NewInt(int64(len(clientCtx.Cfg.P2P.PersistentPeers)))
```

Comment:
This is not true, it does not represents the current connected peers. 
https://docs.ethermint.zone/basics/json_rpc.html#json-rpc-methods

I am not sure if we need Listening and PeerCount at all, did u check if another web3 rpcs like Matic/Binance provide this info?